common/src/stack/command/stack/commands/report/host/__init__.py

```diff
@@ -145,3 +147,23 @@ def run(self, param, args):
 			f.close()
 
 		self.endOutput(padChar='', trimOwner=True)
+
+
+        # Checks if a host interface has atleast one of the following:
+        #       has an aliases
+        #       is on a network with a zone
+        #       has a shortname option
+        #       is the default interface for that host
+        #
+        def validateHostInterface(self, hostname, hostinfo ,aliases):
+                if hostname in aliases and hostinfo['interface'] in aliases[hostname]:
```

Comment:
Are we sure the keys being checked are always going to exist in the hostinfo dict? Are you checking if they exist or if they aren't blanks?

IE: `hostinfo['zone']` vs `'zone' in hostinfo`

Reply:
Actually, looking at the code calling this, I think you've got it right. Looks like the dict should always have those keys.